moveit_setup_assistant/src/widgets/simulation_widget.cpp

```diff
@@ -36,6 +36,8 @@
 // SA
 #include "simulation_widget.h"
 #include "header_widget.h"
+#include "../tools/xml_syntax_highlighter.h"
+#include <moveit/setup_assistant/tools/xml_manipulation.h>
```

Comment:
Why the two different styles of includes?

Reply:
No particular reason. I guess, simply a result of cut-and-paste from [here](https://github.com/ubi-agni/moveit/blob/27ab5a2e13ae58b15b4f0a9d160d0c2dbfcb3c38/moveit_setup_assistant/src/widgets/simulation_widget.cpp).